ospfd/ospf_opaque.c

```diff
@@ -672,6 +675,9 @@ lookup_opaque_info_by_type (struct ospf_lsa *lsa)
           zlog_warn ("Type-11 Opaque-LSA: Reference to OSPF is missing?");
           break; /* Unlikely to happen. */
         }
+      if (IS_DEBUG_OSPF_TRACE)
```

Comment:
Why does only this case get a trace but other paths don't?  What's special here?  Why do we need this?